packages/webdriverio/src/commands/element/isDisplayed.ts

```diff
@@ -59,6 +70,8 @@ export async function isDisplayed (this: WebdriverIO.Element) {
         return false
     }
 
+    const isListedBrowser = noW3CEndpoint.includes((browser.capabilities as WebdriverIO.Capabilities).browserName?.toLowerCase()!) ||
```

Comment:
Let's just use the atom for all cases and remove the complexity it.

Reply:
Doing this, does mean that we no longer support the old Microsoft Edge (non Chromium), that is fine for me but I do want to make sure we are aware.

Reply:
What do you mean by "not supporting it"? It would just use the atom to determine the visibility which hopefully results in the same. That said though, I don't think anyone uses this browser for testing either.